cpu/esp_common/Makefile.features

```diff
@@ -2,6 +2,7 @@
 
 FEATURES_PROVIDED += arch_32bit
 FEATURES_PROVIDED += cpp
+FEATURES_PROVIDED += esp_wifi
```

Comment:
Unrelated?

Also `esp_wifi` isn't really a feature. It is a device driver.

All in all, I don't really understand why this is here.